crates/orchestrator/tests/jobs/da_job/mod.rs

```diff
@@ -0,0 +1,137 @@
+    vec![FieldElement::ONE, FieldElement::ZERO]
+}
+
+#[rstest]
+#[tokio::test]
+async fn test_create_job(
+    #[future] #[from(get_or_init_config)] config: &'static dyn Config,
+) {
+    let config = config.await;
+    let job = DaJob.create_job(config, String::from("0")).await;
+    assert!(job.is_ok());
+
+    let job = job.unwrap();
+    
+    let job_type = job.job_type;
+    assert_eq!(job_type, JobType::DataSubmission);
+    assert_eq!(job.metadata.values().len(), 0, "metadata should be empty");
+    assert_eq!(job.external_id.unwrap_string().unwrap(), String::new(), "external_id should be empty string");
+}
+
+#[rstest]
+// #[should_panic]
+#[tokio::test]
+async fn test_verify_job(
+    #[future] #[from(get_or_init_config)] config: &'static dyn Config,
+    #[from(default_job_item)] job_item: JobItem,
+) {
+    let config = config.await;
+
+    assert!(DaJob.verify_job(config, &job_item).await.is_err());
+}
+
+#[rstest]
+#[should_panic]
+#[tokio::test]
+async fn test_process_job(
+    #[future] #[from(get_or_init_config)] configg: &'static dyn Config,
+    #[from(default_job_item)] job_item: JobItem,
+    state_update: StateUpdate,
+    blob_data: Vec<FieldElement>,
+) {
+    let mut mock_provider = MockMyStarknetProvider::new();
+    // let mut mock_da_client = MockDaClient::new();
+    // let mock_provider = Arc::new(MockMyStarknetProvider::new());
+    let mock_da_client = Arc::new(MockDaClient::new());
+    let mut mock_config = MockConfig::new();
+
+    mock_config.expect_da_client().with().times(1).return_const(Box::new(*Arc::clone(&mock_da_client)));
+    mock_config.expect_starknet_client().with().times(1).returning(move || mock_provider);
+
+
+    // Mocking get_state_update to return a specific state update
+    let expected_block_id = BlockId::Number(123);
+
+    mock_provider.expect_get_state_update()
+        .with(eq(expected_block_id))
+        .times(1)
+        .returning(move |_| 
+            { 
+                Ok( MaybePendingStateUpdate::Update(state_update.clone()))
+            }
+    );
+
+    // Mocking publish_state_diff to return a specific external ID
+    let external_id = "external_id_123".to_string();
+
+    mock_da_client.expect_publish_state_diff()
+        .with(eq(blob_data.clone()))
+        .times(1)
+        .returning(move |_| Ok(external_id.clone()) );
+
+    // Assuming DaJob implements the processing using the Config trait
+    let job_processor = DaJob; // Assuming DaJob is initialized properly
+    let result = job_processor.process_job(&mock_config, &job_item).await;
+
+    assert!(result.is_ok());
+    assert_eq!(result.unwrap(), "external_id_123");
+}
+
+#[rstest]
+fn test_max_process_attempts() {
+    assert_eq!(DaJob.max_process_attempts(), 1);
+}
+
+#[rstest]
+fn test_max_verification_attempts() {
+    assert_eq!(DaJob.max_verification_attempts(), 3);
+}
+
+#[rstest]
+fn test_verification_polling_delay_seconds() {
+    assert_eq!(DaJob.verification_polling_delay_seconds(), 60);
+}
```

Comment:
we can skip these tests. these values will be configurable, we don't want to change the test everytime we change the value